indico/web/templates/_session_bar.html

```diff
@@ -6,175 +6,195 @@
+                    {% pluralize %}
+                        The information on this page is restricted for display on the {{ networks }} networks only.
+                    {% endtrans %}
+                {% elif mode == 'restricted' %}
+                    {% trans %}
+                        The information on this page is restricted for display to named individuals or specific groups.
+                    {% endtrans %}
+                {% endif %}
             </div>
-        {% endif %}
-    </div>
-
-    <button id="protection-details-link"
-       class="arrow i-button protection-{{ mode }} {{ 'icon-shield' if mode != 'public' else 'icon-earth' }}">
-        {% if mode == 'public' %}
-            {% trans %}Public{% endtrans %}
-        {% elif mode == 'network' %}
-            {% trans count=networks|length, networks=networks|join('/') %}
-                {{ networks }} network only
-            {% pluralize %}
-                {{ networks }} networks only
-            {% endtrans %}
-        {% elif mode == 'restricted' %}
-            {% trans %}Restricted{% endtrans %}
-        {% endif %}
-    </button>
+            {% set disclaimer = disclaimers[mode]|sanitize_html|trim %}
+            {% if disclaimer %}
+                <div class="protectionWidgetSection">
+                    {{ disclaimer }}
+                </div>
+            {% endif %}
+        </div>
+    </ind-with-toggletip>
 {% endmacro %}
 
 
 {% macro _render_timezone_selector(timezone_data) %}
     {% set active_tz_label = timezone_data.active_tz %}
+
     {% if timezone_data.disabled %}
-        <span class="i-button icon-time disabled" aria-label="{% trans %}The active timezone is {{ active_tz_label }} and it cannot be changed{% endtrans %}">
+        <span class="icon-time disabled" aria-label="{% trans %}The active timezone is {{ active_tz_label }} and it cannot be changed{% endtrans %}">
             {{- active_tz_label -}}
         </span>
     {% else %}
         {% set profile_timezone = session.user.settings.get('timezone') if session.user else '' %}
-        <article id="tz-selector-widget" style="display:none" aria-labelledby="tz-selector">
-            <div>
-                <h2 id="tz-selector">{% trans %}Choose timezone{% endtrans %}</h2>
 
-                <div id="tz-profile-setting">
-                    <span>{% trans %}Your profile timezone:{% endtrans %}</span>
-                    <span>{{ profile_timezone }}</span>
-                </div>
+        <ind-with-popup>
+            <button id="tz-selector-link" data-trigger class="popup-trigger">
+                <span class="tz-selector-label">{% trans %}Page timezone:{% endtrans %}</span>
+                {{ timezone_data.active_tz }}
+            </button>
+            <dialog data-dialog>
+                <article id="tz-selector-widget" aria-labelledby="tz-selector">
+                    <div>
+                        <h2 id="tz-selector">{% trans %}Choose timezone{% endtrans %}</h2>
```

Comment:
The bold still looks quite bad there. Using `--text-color` for it instead of hard black makes it much nicer.

Likewise for the labels such as "Your profile timezone", "Use timezone based on:" and "Remember these settings"